content/docs/command-reference/push.md

```diff
@@ -65,18 +65,18 @@ cache (compared to the default remote.) It can be used to see what files
 `dvc push` would upload.
 
 If one or more `targets` are specified, DVC only considers the files associated
-with those DVC-files. Using the `--with-deps` option, DVC tracks dependencies
+with those `.dvc` files. Using the `--with-deps` option, DVC tracks dependencies
 backward from the target [stage files](/doc/command-reference/run), through the
 corresponding [pipelines](/doc/command-reference/pipeline), to find data files
 to push.
 
 ## Options
 
-- `-a`, `--all-branches` - determines the files to upload by examining DVC-files
-  in all Git branches instead of just those present in the current workspace.
-  It's useful if branches are used to track experiments or project checkpoints.
-  Note that this can be combined with `-T` below, for example using the `-aT`
-  flag.
+- `-a`, `--all-branches` - determines the files to upload by examining `.dvc`
```

Comment:
All the same changes requested for pull apply here too.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/1408#pullrequestreview-427734039